css/overrides.css

```diff
@@ -347,3 +347,39 @@ span.password-toggle-wrapper {
 .cyborg .menu-dropdown a {
   color: #888888;
 }
+
+/**
+ * @file
+ * Card listing view on the default home page.
+ */
+
+.block-views-promoted-cards-block .views-view-grid-cols-3 {
```

Comment:
We could change this (and same below) to just `.views-view-grid-cols-3` (without the `.block-views-promoted-cards-block`) to override the CSS in Views.

Reply:
done